packages/client/wallets/aa/src/blockchain/token/Tokens.ts

```diff
@@ -25,3 +29,54 @@ export type ERC20TransferType = { token: ERC2OEVMToken; amount: bigint };
 export type SFTTransferType = { token: SFTEVMToken; quantity: number };
 export type NFTTransferType = { token: NFTEVMToken };
 export type TransferType = ERC20TransferType | SFTTransferType | NFTTransferType;
+
+export function transferParams({
+    contract,
+    config,
+    from,
+    to,
+}: {
+    from: Account;
+    contract: Address;
+    to: Address;
+    config: TransferType;
+}): {
+    account: Account;
+    address: Address;
+    abi: any;
+    functionName: string;
+    args: any[];
+    tokenId?: string;
+} {
```

Comment:
nit: this function signature is pretty unwieldy–some types could help `TransferInput`, `TransferTransaction`/`TransferParamsOutput`
naming a bit tough on this one

Reply:
good point, done